<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FLCreateFloorDlg</class>
 <widget class="QDialog" name="FLCreateFloorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create new floor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Floor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Floor Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="leName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Floor Graph</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="leGraph">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Choose Graph</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Floor Size</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lbl_floor_width">
        <property name="text">
         <string>Width = 0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLabel" name="lbl_floor_height">
        <property name="text">
         <string>Height = 0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cb_scale">
        <item>
         <property name="text">
          <string>1:1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:1000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:2000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1:5000</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Unit</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Meters</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yards</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Feet</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Inches</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Centimeters</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Comments</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Zone Controllor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="4">
       <widget class="FLTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pbAdd">
        <property name="text">
         <string>Add ZC</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pbDelete">
        <property name="text">
         <string>Delete ZC</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pbEdit">
        <property name="text">
         <string>Edit ZC</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>52</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>242</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FLTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>fltreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
